### ISSUE TITLE
Add content to linked pages (other than home page)

### ISSUE THREAD

**Issue by apoorva-mk** [acm-w-nitk/acm-w-nitk.github.io#61]
- Replace dummy text with content.
- The content shouldn't overlap with the one on the main page. 
- Add the remaining content to the relevant page.
- Only add it in paragraphs (no bullets)
- Ensure it is visible clearly and doesn't get hidden by the background.

Please find the content [here](https://docs.google.com/document/d/10fHijD0sk2z5UVdTzEglzq78QkYrurD38f_oL2OjCtY/edit)

**Comment by thewhiteswan** [comment]
Hi @apoorva-mk I would be happy to take on this issue!

I also just want to make sure I understand "content shouldn't overlap with the one on the main page". 

For example, the "Discover Opportunities" content in the doc only has one extra line that doesn't already appear on the main page: "Get inspired and be a part of empowering women." Is this all you would want to be added to the "Discover" page?

**Comment by apoorva-mk** [comment]
> Hi @apoorva-mk I would be happy to take on this issue!
> 
> I also just want to make sure I understand "content shouldn't overlap with the one on the main page".
> 
> For example, the "Discover Opportunities" content in the doc only has one extra line that doesn't already appear on the main page: "Get inspired and be a part of empowering women." Is this all you would want to be added to the "Discover" page?

Hi, @thewhiteswan thanks for pointing it out. I think you can just update it, for now but it would be great if you could maybe rephrase the other parts.

**Comment by thewhiteswan** [comment]
Gotcha! I'll give it a go :)

**Comment by adi-rebellion** [comment]
Hello Mam,
Don't you think the UI of the website should be changed to create a better user interaction with the website

**Comment by apoorva-mk** [comment]
> Hello Mam,
> Don't you think the UI of the website should be changed to create a better user interaction with the website

Hi, @adi-rebellion raise it as separate issue. Please look through our guidelines before starting.
